guides/common/modules/snip_content-types-export.adoc

```diff
@@ -0,0 +1,10 @@
+You can export the following content from {ProjectServer}:
+
+* Ansible repositories
```

Comment:
When we're at it, can we also do something like this?
```suggestion
* Ansible collections content
```
The content type for Ansible is actually called "ansible collection" in the web UI.